pandas/core/generic.py

```diff
@@ -842,7 +842,39 @@ def __contains__(self, key):
 
     @property
     def empty(self):
-        """True if NDFrame is entirely empty [no items]"""
+        """True if NDFrame is entirely empty [no items], i.e. any of the axes
+        are of length 0.
+
+        Notes
```

Comment:
I think it is OK to leave it here (Jeff, in the html rendered version, the notes will also be before the examples, irregardless of how the order is here)